sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1548,6 +1821,82 @@ submitted in its entirety for execution via
   ...
 ----
 
+=== Dynamic Parameter Update
+
+Example showing a graph with a single kernel node that is created using a kernel
+bundle with `handler::set_args()` and having its node arguments updated.
+
+[source,c++]
+----
+...
+
+using namespace sycl;
+namespace sycl_ext = sycl::ext::oneapi::experimental;
+
+queue myQueue;
+auto myContext = myQueue.get_context();
+auto myDevice = myQueue.get_device();
+
+// USM allocations for kernel input/output
+const size_t n = 1024;
+int *ptrX = malloc_shared<int>(n, myQueue);
+int *ptrY = malloc_device<int>(n, myQueue);
+
+int *ptrZ = malloc_shared<int>(n, myQueue);
+int *ptrQ = malloc_device<int>(n, myQueue);
+
+// Kernel loaded from kernel bundle
+const std::vector<kernel_id> builtinKernelIds =
+      myDevice.get_info<info::device::built_in_kernel_ids>();
+kernel_bundle<bundle_state::executable> myBundle =
+      get_kernel_bundle(myContext, { myDevice }, builtinKernelIds);
+kernel builtinKernel = myBundle.get_kernel(builtinKernelIds[0]);
+
+// Graph containing a two kernels node
+sycl_ext::command_graph myGraph(myContext, myDevice);
+
+// Create graph dynamic parameters
+dynamic_parameter<int*> dynParamInput(myGraph);
```

Comment:
Did void* support? If we did not know the parameter type.
Like:
`dynamic_parameter<void*> dynParamInput(myGraph);`

Reply:
I think this would only work for USM arguments